Controller/Adminhtml/Menu/Index.php

```diff
@@ -1,29 +1,19 @@
 <?php
+declare(strict_types=1);
+
 namespace Snowdog\Menu\Controller\Adminhtml\Menu;
 
 use Magento\Backend\App\Action;
 use Magento\Backend\App\Action\Context;
+use Magento\Framework\Controller\ResultFactory;
 use Magento\Framework\View\Result\PageFactory;
```

Comment:
`Context` and `PageFactory` are not used anymore so I think they can be safely removed from here.

Reply:
Yes, you are right. I'm going to adjust that later afternoon.